quick_start.sh

```diff
@@ -65,11 +96,12 @@ then
 else
     source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/Flowminder/FlowKit/${BRANCH:-master}/development_environment)"
     echo "Starting containers"
```

Comment:
When I tried spinning up the quick-start examples, for a while the script just seemed to be hanging. Somehow docker-compose doesn't give any visual indication that it's downloading the images. Can we add a comment to this effect here? E.g. add: "this may take a while" (and perhaps even indicate that it may look like nothing is happening for a while)?

Reply:
That's weird.. I definitely get output from compose during the pull and the up.

Reply:
Yes, I do get output, but it is static and doesn't change for a while until the pull is complete.

It basically looks like this for a couple of minutes:
```
Starting containers
Pulling flowdb                   ...
Pulling flowdb_testdata          ...
Pulling flowdb_synthetic_data    ...
Pulling worked_examples          ...
Pulling flowapi                  ...
Pulling flowauth                 ...
Pulling flowmachine_query_locker ...
Pulling flowmachine              ...
```
But since there is no visual interaction and nothing changes, it's easy to think that the script is hanging or something went wrong (until the next bit of output appears after a while).

How about changing the message to the following?
```suggestion
     echo "Starting containers (this may take a few minutes)"
```